docs/secure-log-path-variable.md

```diff
@@ -14,6 +12,87 @@ Implemented in Percona Server for MySQL 8.0.28-19 (2022-05-12).
 | Data type     | String            |
 | Default       | empty string      |
 
-This variable restricts the dynamic log file locations. The variable is read-only and must be set up in a configuration file or the command line.
+This variable restricts the dynamic log file locations. The variable is read-only and is set up in a configuration file or the command line.
+
+The accepted value is the directory name as a string. The default value is an empty string. When the value is an empty string, the variable only adds a warning to the error log and does nothing and the slow quary log and the general log are located in the default, /var/lib/mysql, directory. If the value contains a directory name, then the slow query log and the general log are located in that directory. An attempt to move either of these files outside of the specified directory results in an error.
+
+## The example of the secure_log_path variable usage
+
+Run the following commands as root:
+
+1. Create the direcory to store the slow query log and the general log.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
+    [root@localhost ~]# mkdir /var/lib/mysqld-logs
+    ```
+
+2. Set up the `secure_log_path` variable with the created directory in a configuration file.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
+    [root@localhost ~]# echo -en "secure_log_path = /var/lib/mysqld-logs\n" >> /etc/my.cnf
+    ```
+
+3. Changes the owner and group of the `/var/lib/mysqld-logs` directory and all its subdirectories and files to `mysql`.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
+    [root@localhost ~]# chown -R mysql:mysql /var/lib/mysqld-logs
+    ```
+
+4. Restart the MySQL server.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
+    [root@localhost ~]# systemctl restart mysql
+    ```
+
+5. Check that the `/var/lib/mysqld-logs/` directory is set for the `secure_log_path` variable.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
+    [root@localhost ~]# mysql -e "select @@secure_log_path"
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        +-----------------------+
+        | @@secure_log_path     |
+        +-----------------------+
+        | /var/lib/mysqld-logs/ |
+        +-----------------------+
+        ```
+
+6. Enable the slow query log and the general log for the MySQL server.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
+    [root@localhost ~]# mysql -e "set global slow_query_log=1"
+    [root@localhost ~]# mysql -e "set global general_log=1"
+    ```
+
+7. Check that the slow query log and the general log are enabled for the MySQL server.
```

Comment:
This example will not work now. It will not allow to enable slow query log and general log without previously updating slow and general log paths to be in secure_log_path directory. It may be better to modify this example to be similar to this test case - https://github.com/percona/percona-server/pull/5119/files#diff-060c36fbec241655c05f0b85c8dc25b38696c34ba1031d9b58e0703b4f28ac34R13.

1) Set everything up in config file first:
    secure_log_path=/var/lib/mysqld-logs
    general-log=ON
    general-log-file=/var/lib/mysqld-logs/general_log
    slow-query-log=ON
    slow-query-log-file=/var/lib/mysqld-logs/slow_log
    buffered-error-log-size=1000
    buffered-error-log-filename=/var/lib/mysqld-logs/buffered_log

2) restart server
3) do some checkups to verify everything works as expected

Reply:
Updated